src/app/core/admin/admin-entity-details/admin-entity-form/admin-entity-form.component.ts

```diff
@@ -64,6 +64,9 @@ import { FieldGroup } from "app/core/entity-details/form/field-group";
 })
 export class AdminEntityFormComponent implements OnChanges {
   @Input() entityType: EntityConstructor;
+  // tracking the tab index and section index for the connected drop areas for drag&drop
+  @Input() tabIndex: number;
+  @Input() sectionIndex: number;
```

Comment:
I think at the level of (admin-)entity-form, it doesn't make a difference that there are tabs and sections. let's hide that detail/complexity here in the component and just take one "sectionID" (or something along those lines) as input?

Reply:
yes, that makes sense now. I’m now using a single input called **uniqueAreaId**, which holds both the sectionIndex and tabIndex.